pkg/fixtures/fixtures.go

```diff
@@ -254,6 +262,23 @@ func (fxt *Fixture) Remove(removals []string) error {
 	return nil
 }
 
+// Edit opens the fixture in the git's default IDE to edit directly
+func (fxt *Fixture) Edit(path string, filedata []byte) ([]byte, error) {
+	filename := getFixtureFilenameWithWildcard(path)
+	editor, err := git.NewEditor(filename, filedata)
+	if err != nil {
+		return nil, fixtureRewriteError{operation: "edit", err: err, fixture: fxt}
+	}
+
+	return editor.EditContent()
+}
+
+func getFixtureFilenameWithWildcard(path string) string {
+	pathComponents := strings.Split(path, "/")
+	fixtureName := strings.Split(pathComponents[len(pathComponents)-1], ".")
+	return strings.Join(fixtureName[0:len(fixtureName)-1], ".") + ".*." + fixtureName[len(fixtureName)-1]
```

Comment:
why do we need to add wildcard to the fixtures path name?

Reply:
why cant we have the random string appended to the end? why is in the middle a better?

Reply:
appending the string at the end (after the `.json` file extension) gets rid of syntax highlighting for JSON file in your editor